### PR TITLE
[FR] Beatmap Editor/AiMod

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the AiMod page

### CODE REVIEW

**Comment by bdach** [comment]
It would probably have been more productive to ask someone to reopen the old PRs since they seem to have the exact same contents, extrapolating from comparing this to #3023. But then again I haven't checked all of them.

**Comment by DoryanR** [comment]
It's true, it would have been simpler, but I'd rather do it again, it will be better for me.

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# AiMod
+
+**AiMod** est un outil de modding intégré a [L'éditeur de Beatmaps](/wiki/Beatmap_Editor). Pour ouvrir AiMod, pressez `Control` + `Shift` + `A` ou naviguez via les options du menu, **File** » **Ouvrir AiMod** une fois qu'une beatmap est ouverte dans l'éditeur de beatmap.
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
**AiMod** est un outil de modding intégré a l'[éditeur de Beatmaps](/wiki/Beatmap_Editor). Pour ouvrir AiMod, pressez `Control` + `Shift` + `A` ou naviguez via les options du menu, **File** » **Ouvrir AiMod** une fois qu'une beatmap est ouverte dans l'éditeur de beatmap.
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+*Remarque: Les nombres entre crochets (par exemple `{0}`) sont des espaces réservés pour un nombre ou un mot.*
+
+### Informatif
+
+#### Tous
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| Aucun problème n'a été trouvé sur cette carte ! | AiMod n'a trouvé aucun problème avec votre carte ! | Vous pouvez télécharger votre beatmap à l'aide de BSS pour permettre aux osu!modders de trouver d'autres erreurs et problèmes avec votre beatmap. |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Aucun problème n'a été trouvé sur cette map ! | AiMod n'a trouvé aucun problème avec votre map ! | Vous pouvez télécharger votre beatmap à l'aide de BSS pour permettre aux osu!modders de trouver d'autres erreurs et problèmes avec votre beatmap. |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+#### Meta
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| Le taux de HP pour Easy/Normal devrait être d'au moins 4. | Cartes osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez le taux de HP sur 4 ou plus. |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le taux de HP pour Easy/Normal devrait être d'au moins 4. | Maps osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez les HP sur 4 ou plus. |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+| Le taux de HP pour Hard et plus devrait être d'au moins 7. | Cartes osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez le taux de HP sur 7 ou plus. |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le taux de HP pour Hard et plus devrait être d'au moins 7. | Maps osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez les HP sur 7 ou plus. |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+| Le taux de OD pour les cartes avec peu de curseurs devrait être d'au moins 7. | Cartes osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez le taux OD sur 7 ou plus. |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le taux de OD pour les maps avec peu de curseurs devrait être d'au moins 7. | Maps osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez l'OD sur 7 ou plus. |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+| La vitesse du curseur doit être de 1,40 ou 1,60. | Cartes osu!taiko uniquement ! | Accédez à l'onglet Timing et modifiez la vitesse du curseur sur 1,40 ou 1,60 |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| La vitesse du curseur doit être de 1,40 ou 1,60. | Maps osu!taiko uniquement ! | Accédez à l'onglet Timing et modifiez la vitesse du curseur sur 1,40 ou 1,60 |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+| Votre beatmap n'a pas d'image d'arrière-plan. | La difficulté actuelle n'a pas d'image d'arrière-plan ! Notez que les vidéos ne comptent pas comme images d'arrière-plan car l'utilisateur n'est pas obligé de les télécharger | Trouvez une image appropriée et utilisez-la comme image d'arrière-plan. |
+
+### Avertissements
+
+#### Compose
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| Ce curseur se déplace de manière anormale. | | |
+| Ce combo est très long. Pensez à le diviser. | La longueur des combos peut affecter la difficulté de la carte, ainsi que la quantité de fruits qui seront empilés sur l'assiette du receveur. | Accédez aux combos incriminés et décidez par où commencer le nouveau combo. (Il est préférable que la durée des combos soit au maximum de 15 à 18) |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce combo est très long. Pensez à le diviser. | La longueur des combos peut affecter la difficulté de la map, ainsi que la quantité de fruits qui seront empilés sur l'assiette du receveur. | Accédez aux combos incriminés et décidez par où commencer le nouveau combo. (Il est préférable que la durée des combos soit au maximum de 15 à 18) |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+| Ce spinner apparaît à l'écran plus tard que les objets qui le suivent. | Les objets apparaissent alors que le spinner est actif. Cela peut être dû au fait que AR est faible et/ou en plaçant des objets trop près (dans le temps) du spinner. | Modifiez la longueur du spinner et/ou supprimez les objets suivants |
+| La fin de l'objet est hors écran ! | Les objets hors écran peuvent ne pas être visibles dans les clients osu! Exécutés dans des proportions d'un ratio 4:3. | Supprimer ou déplacer l'extrémité de l'objet |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| La fin de l'objet est hors de l'écran ! | Les objets hors de l'écran peuvent ne pas être visibles dans les clients osu! Exécutés dans des proportions d'un ratio 4:3. | Supprimer ou déplacer l'extrémité de l'objet |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+| L'objet est hors écran ! | Les objets hors écran peuvent ne pas être visibles dans les clients osu! Exécutés dans des proportions d'un ratio 4:3. | Supprimer ou déplacer l'objet |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| L'objet est hors de l'écran ! | Les objets hors de l'écran peuvent ne pas être visibles dans les clients osu! Exécutés dans des proportions d'un ratio 4:3. | Supprimer ou déplacer l'objet |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+| Cet objet est trop proche de l'objet précédent. | | |
+| Cet objet est trop éloigné de l'objet précédent. | | |
+| Ce spinner est trop court. Auto doit atteindre au moins 1000 points bonus sur les spinners. | Le spinner spécifié est trop court; cela rendra impossible l'obtention d'un SS. | Trouvez le spinner et retirez-le ou rallongez-le un peu plus longtemps sur 3 temps entiers. |
+| Ces deux objets existent au même moment ! | Deux objets sont superposés. Cela rendra la carte impossible à classer ! | Trouvez les deux objets touchés et déplacez-les ou supprimez-les. |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ces deux objets existent au même moment ! | Deux objets sont superposés. Cela rendra la map impossible à classer ! | Trouvez les deux objets au même moment et déplacez-les ou supprimez-les. |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+| Cet objet chevauche un autre objet. | Cartes osu!mania uniquement ! | |
+| Cet objet est empilé sur un autre objet. | Cartes osu!mania uniquement ! | Les touches et/ou les notes spécifiées sont superposées. Cela rendra votre carte impossible d'obtenir un SS ! | Trouvez la clé ou maintenez la note et corrigez-la ou supprimez-la. |
+| Plus de 6 notes simultanément ne sont pas autorisées. | Cartes osu!mania uniquement ! | La plupart des claviers peuvent autoriser jusqu'à 6 pressions de touches à la fois. | Vérifiez votre beatmap pour vous assurer que chaque colonne a au plus 6 pressions de notes. |
+
+#### Design
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| L'image d'arrière-plan est plus grande que 2560x1440. | | |
+| Cette carte peut nécessiter un avertissement d'épilepsie, car elle contient des story-boards fréquemment basculés. | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Cette map peut nécessiter un avertissement d'épilepsie, car elle contient des story-boards fréquemment basculés. | | |
```

**Comment by DoryanR** [comment]
@almaloste Quand tu aura fini le reste du fichier je pourrait corriger, c'est mieux de tout faire en même temps.

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+| Les dimensions {0} doivent être {1}x{1} | | |
+| Les dimensions de votre vidéo ne doivent pas dépasser 1024x768 pour le format en ratio 4:3. | | |
+| Les dimensions de votre vidéo ne doivent pas dépasser 1280x720 pour le format en ratio 16:9. | | |
+
+#### Timing
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| Toutes les sections de chronométrage ont un volume inférieur à 5%. | | |
+| Cette beatmap dure plus de 6 minutes. Envisagez de la raccourcir si ce n'est pas une carte de style marathon. | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Cette beatmap dure plus de 6 minutes. Envisagez de la raccourcir si ce n'est pas une map de style marathon. | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+| Votre beatmap est inférieure à 45 secondes. Pensez à l'allonger. | | |
+| Le débit binaire audio est supérieur à 192 kbps. Pensez à recompresser en CBR 192 kbps ou VBR ~ 1.0. | | |
+| Le temps de Kiai est activé pendant moins de 15 secondes. | | |
+| Le débit binaire audio est inférieur à 128 kbps. Pensez à trouver une source de meilleure qualité. | | |
+| Le mp3 que vous utilisez est beaucoup plus long que la partie mappée. Pensez à le recadrer pour réduire la taille du fichier. | | |
+| Kiai a besoin d'un point de fin. | | |
+| Un point d'aperçu pour cette carte n'est pas défini. Pensez à en définir un dans le menu Timing. | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Un point d'aperçu pour cette map n'est pas défini. Pensez à en définir un dans le menu Timing. | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+| Deux points de synchronisation existent en même temps ! | | |
+| {0} est hors de {1}, les sections de chronométrage ont un volume inférieur à 5%. | | |
+| Plus du tiers de la carte correspond à l'heure de Kiai. Pensez à réduire cela. | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Plus du tiers de la carte correspond au Kiai time. Pensez à réduire cela. | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+| Plus de la moitié de la carte de taille TV correspond à l'heure de Kiai. Pensez à réduire cela. | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Plus de la moitié de la carte de taille TV correspond au Kiai time. Pensez à réduire cela. | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+| Kiai n'est pas cassé ! | | |
+| La fin de Kiai n'est pas cassée ! | | |
+| Le temps de pause n'est pas suggéré pour les cartes mania | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le temps de pause n'est pas suggéré pour les maps mania | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 17]
```diff
@@ -0,0 +1,156 @@
...
+| Le diff Easy/Normal contient trop de changements de vitesse. | | |
+| Kiai est basculé très fréquemment ! | | |
+| Plus du tiers de la carte correspond à l'heure de Kiai. Pensez à réduire cela. | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]: Pourquoi cette phrase est notée 4 fois ?
```suggestion
| Plus du tiers de la map correspond au Kiai time. Pensez à réduire cela. | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 18]
```diff
@@ -0,0 +1,156 @@
...
+
+#### Meta
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| La clémence de la pile est supérieure à 0,9 ou inférieure à 0,3. | | |
+| L'artiste romanisé contient unicode. | | |
+| Le titre romanisé contient unicode. | | |
+| Le compte à rebours n'est pas autorisé en mode mania. | Cartes osu!mania uniquement ! | Allez dans Paramètres de la chanson et désactivez le compte à rebours dans l'onglet "Design" |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le compte à rebours n'est pas autorisé en mode mania. | Maps osu!mania uniquement ! | Allez dans Paramètres de la chanson et désactivez le compte à rebours dans l'onglet "Design" |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 19]
```diff
@@ -0,0 +1,156 @@
...
+| La boîte aux lettres n'est pas autorisée en mode mania. | Cartes osu!mania uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| La boîte aux lettres n'est pas autorisée en mode mania. | Maps osu!mania uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 20]
```diff
@@ -0,0 +1,156 @@
...
+| Le taux de OD devrait être d'au moins 5. | Cartes osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez le taux OD sur 5 ou plus |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le taux de OD devrait être d'au moins 5. | Maps osu!mania uniquement ! | Accédez aux paramètres de la chanson et définissez le taux OD sur 5 ou plus |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 21]
```diff
@@ -0,0 +1,156 @@
...
+| Le taux de OD pour les cartes avec très peu de curseurs devrait être d'au moins 8. | Cartes osu!mania uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le taux de OD pour les maps avec très peu de curseurs devrait être d'au moins 8. | Maps osu!mania uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 22]
```diff
@@ -0,0 +1,156 @@
...
+| Le compte à rebours n'est pas autorisé en mode taiko. | Cartes osu!taiko uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Le compte à rebours n'est pas autorisé en mode taiko. | Maps osu!taiko uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 23]
```diff
@@ -0,0 +1,156 @@
...
+| L'avertissement d'épilepsie n'est pas autorisé en mode taiko. | Cartes osu!taiko uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| L'avertissement concernant l'épilepsie n'est pas autorisé en mode taiko. | Maps osu!taiko uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 24]
```diff
@@ -0,0 +1,156 @@
...
+| La boîte aux lettres n'est pas autorisée en mode taiko. | Cartes osu!taiko uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| La boîte aux lettres n'est pas autorisée en mode taiko. | Maps osu!taiko uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 25]
```diff
@@ -0,0 +1,156 @@
...
+| Le compte à rebours est en conflit avec la différence {0}. | | |
+| L'entrée audio est en conflit avec la différence {0}. | | |
+| La boîte aux lettres dans les ruptures est en conflit avec la différence {0} | | |
+| Le temps de prévisualisation audio est en conflit avec la différence {0}. | | |
+| La source est en conflit avec la différence {0}. | | |
+| Les balises sont en conflit avec la différence {0}. | | |
+| Le titre est en conflit avec la différence {0} | | |
+| L'artiste Unicode est en conflit avec la différence {0}. | | |
+| Le titre Unicode est en conflit avec la différence {0}. | | |
+| Ce jeu de cartes ne peut pas avoir une diff folle sans une diff dure. | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce jeu de maps ne peut pas avoir une diff folle sans une diff dure. | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 26]
```diff
@@ -0,0 +1,156 @@
...
+| La Beatmap dépasse la taille de téléchargement autorisée ({0}kb autorisé) | Le zippé en `.osz` est trop large | |
+| Ce jeu de cartes n'a pas de diff : {0} | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce jeu de maps n'a pas de diff : {0} | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 27]
```diff
@@ -0,0 +1,156 @@
...
+| Ce jeu de cartes a besoin d'au moins 2 différences osu!catch. | Cartes osu!catch uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce jeu de maps a besoin d'au moins 2 différences osu!catch. | Cartes osu!catch uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 28]
```diff
@@ -0,0 +1,156 @@
...
+| Ce jeu de cartes a besoin d'au moins 1 différence Easy/Normal | | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce jeu de maps a besoin d'au moins 1 différence Easy/Normal | | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 29]
```diff
@@ -0,0 +1,156 @@
...
+| Ce jeu de cartes a besoin d'au moins 2 différence osu!mania . | Cartes osu!mania uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce jeu de maps a besoin d'au moins 2 différence osu!mania . | Maps osu!mania uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 30]
```diff
@@ -0,0 +1,156 @@
...
+| Ce jeu de cartes nécessite au moins 2 différences standard. | Cartes standard uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce jeu de maps nécessite au moins 2 différences standard. | Maps standard uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 31]
```diff
@@ -0,0 +1,156 @@
...
+| Ce jeu de cartes nécessite au moins 2 différences osu!taiko. | Cartes osu!taiko uniquement ! | |
```
Comment (almaloste) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
| Ce jeu de maps nécessite au moins 2 différences osu!taiko. | Maps osu!taiko uniquement ! | |
```

File: wiki/Beatmap_Editor/AiMod/fr.md (hunk 17 above)
Comment (DoryanR) [reply wiki/Beatmap_Editor/AiMod/fr.md]: Aucune idée, elle est comme ça dans le fichier en.md d'origine

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 32]
```diff
@@ -0,0 +1,156 @@
+# AiMod
+
+est un outil de modding intégré a l'[éditeur de Beatmaps](/wiki/Beatmap_Editor). Pour ouvrir AiMod, pressez `Control` + `Shift` + `A` ou naviguez via les options du menu, **File** » **Ouvrir AiMod** une fois qu'une beatmap est ouverte dans l'éditeur de beatmap.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/AiMod/fr.md]:
```suggestion
Est un outil de modding intégré a l'[éditeur de Beatmaps](/wiki/Beatmap_Editor). Pour ouvrir AiMod, pressez `Control` + `Shift` + `A` ou naviguez via les options du menu, **File** » **Ouvrir AiMod** une fois qu'une beatmap est ouverte dans l'éditeur de beatmap.
```

?

File: wiki/Beatmap_Editor/AiMod/fr.md [hunk 33]
```diff
@@ -0,0 +1,156 @@
...
+| Le titre Unicode est en conflit avec la différence {0}. | | |
+| Ce jeu de maps ne peut pas avoir une diff folle sans une diff dure. | | |
+| La Beatmap dépasse la taille de téléchargement autorisée ({0}kb autorisé) | Le zippé en `.osz` est trop large | |
+| Ce jeu de maps n'a pas de diff : {0} | | |
+| Ce jeu de maps a besoin d'au moins 2 différences osu!catch. | Cartes osu!catch uniquement ! | |
+| Ce jeu de maps a besoin d'au moins 1 différence Easy/Normal | | |
+| Ce jeu de maps a besoin d'au moins 2 différence osu!mania . | Maps osu!mania uniquement ! | |
+| Ce jeu de maps nécessite au moins 2 différences standard. | Maps standard uniquement ! | |
+| Ce jeu de maps nécessite au moins 2 différences osu!taiko. | Maps osu!taiko uniquement ! | |
+| Les points de synchronisation non hérités sont en conflit avec la différence {0} | | |
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/AiMod/fr.md]: It looks like you are missing a empty line at the end, please add it in.

**Pull request merged**